rcl/src/rcl/wait.c

```diff
@@ -236,7 +283,7 @@ rcl_wait_set_get_allocator(const rcl_wait_set_t * wait_set, rcl_allocator_t * al
   memset( \
     wait_set->impl->RMWStorage, \
     0, \
-    sizeof(rmw_subscription_t *) * wait_set->impl->RMWCount);
+    sizeof(rmw_ ## Type ## _t *) * wait_set->impl->RMWCount);
```

Comment:
Hmm, good catch. I guess it didn't show up in testing since it's always the sizeof a pointer, but still a good thing to fix.